tests/network_test.cpp

```diff
@@ -136,34 +136,30 @@ TEST_F(NetworkTest, send_pbft_block) {
                  [&](auto& ctx) { WAIT_EXPECT_EQ(ctx, nw1->getPeer(node2_id)->pbft_chain_size_, chain_size) });
 }
 
-// Test creates two Network setup and verifies sending transaction
-// between is successfull
+// Test creates two Network setup and verifies sending transaction between is successfull
 TEST_F(NetworkTest, transfer_transaction) {
-  std::unique_ptr<Network> nw1(new taraxa::Network(g_conf1->network));
-  std::unique_ptr<Network> nw2(new taraxa::Network(g_conf2->network));
-
+  auto nw1 = u_ptr(new Network(g_conf1->network));
+  auto nw2 = u_ptr(new Network(g_conf2->network));
```

Comment:
when you refactor old code, please lets do not copy paste things like, use make_unique or make_shared instead of new... and get rid of u_ptr too...